backend/src/modules/tournament-tables/core/domain/TournamentTable.ts

```diff
@@ -109,14 +115,18 @@ export function releaseTournamentTable(
 
   const releasedTable = onTableAvailabilityWasChanged(state);
 
-  const tournamentTableWasReleased = new TournamentTableWasReleased({
-    occurredAt: currentTime,
-    tournamentId: releasedTable.tournamentId,
-    tableNumber: releasedTable.tableNumber.raw,
-  });
+  const tournamentTableWasReleasedEvent = tournamentTableWasReleased(
+    currentTime,
+    releasedTable.tournamentId,
+    releasedTable.tableNumber.raw,
+  );
 
   return {
     state: releasedTable,
-    events: [tournamentTableWasReleased],
+    events: [tournamentTableWasReleasedEvent],
   };
 }
+
+function tournamentTableWasReleased(occurredAt: Date, tournamentId: string, tableNumber: number): TournamentTableWasReleased {
+  return new TournamentTableWasReleased({ occurredAt, tournamentId, tableNumber });
+}
```

Comment:
ok
Dlaczego masz  konstruktor wrzucony do osobnej funkcji?
Jaka jest zaleta takiego rozwiązania?

Reply:
Używam tego teraz też w assignTablesToTournament, minimalnie upraszcza zapis.

Reply:
Jak dla mnie wlasnie przeszkadza, ze teraz nie masz nazwyanych parametrow ;)

Reply:
ok, to zmienię